corehq/apps/ota/views.py

```diff
@@ -59,7 +59,7 @@ def get_restore_response(domain, couch_user, app_id=None, since=None, version='1
                             status=401)
 
     project = Domain.get_by_name(domain)
-    app = Application.get_db().get(app_id) if app_id else None
+    app = Application.get(app_id) if app_id else None
```

Comment:
that's right. actually looking closer I think this should be [`get_app`](https://github.com/dimagi/commcare-hq/blob/b9ee8fac2949ee7d0d84633d8367ae8d8479a6e7/corehq/apps/app_manager/dbaccessors.py#L57) since we would also want to not crash on `RemoteApp` restores